BasicSample/Assets/ARAnchor/Scripts/AnchorsSample.cs

```diff
@@ -48,7 +48,7 @@ async void Start()
             }
             m_arAnchorManager.anchorsChanged += AnchorsChanged;
 
-            m_anchorStore = await XRAnchorStore.LoadAsync(m_arAnchorManager.subsystem);
+            m_anchorStore = await m_arAnchorManager.LoadAnchorStoreAsync();
```

Comment:
also just curious how did pipeline have a successful basic sample build in CI without this change?

Reply:
This is a warning instead of error.